epsagon/wrappers/fastapi.py

```diff
@@ -248,21 +247,27 @@ def wrapped_handler(*args, **kwargs):
     dependant.call = wrapped_handler
 
 
-class TracingAPIRoute(APIRoute):
+def route_class_wrapper(wrapped, instance, args, kwargs):
     """
-    Custom tracing route - traces each route request & response
+    Route class wrapper - traces each route request & response.
+    :param wrapped: wrapt's wrapped
+    :param instance: wrapt's instance
+    :param args: wrapt's args
+    :param kwargs: wrapt's kwargs
     """
+    result = wrapped(*args, **kwargs)
+    # Skip on Lambda environment since it's not relevant and might be duplicate
+    if not is_lambda_env() and instance:
+        try:
+            if instance.dependant and instance.dependant.call:
+                _wrap_handler(
+                    instance.dependant,
+                    kwargs.get('status_code', DEFAULT_SUCCESS_STATUS_CODE)
```

Comment:
Maybe the default status code should be UNKNOWN or something that will differentiate it from cases where we could get the status code and it was successful?

Reply:
its actually our default value to match the FastAPI default endpoint status code